src/services/news/index.js

```diff
@@ -349,6 +350,7 @@ class NewsService extends AbstractService {
 				$or: subqueries,
 				$limit: query.$limit,
 				$skip: query.$skip,
+				$sort: query.$sort, // sortQuery do not work as expect and need in client
```

Comment:
I don't like this. There is logic for handling the sort parameter, why not use it? IMO, you just have to rename `$sort` to `sort` in the client and you're good (see line 341).

Reply:
I change the request to sort=-displayAt